STL_Extension/include/CGAL/Base_with_time_stamp.h

```diff
@@ -18,9 +18,11 @@ namespace CGAL {
 
 template <typename Base>
 class Base_with_time_stamp : public Base {
-  std::size_t time_stamp_ = -1;
+  std::size_t time_stamp_ = 0;
```

Comment:
I can't remember why we made that decision, but IIRC the `-1` value was there on purpose, used as the "non initialized" time stamp value

Reply:
After a discussion IRL with Jane, we decided that the time stamps in that class should be internalized to another sentinel value (such as `-2`), and add assertions to check that during runtime the time stamps are not equal to `-1` or `-2`.

Reply:
now fixed in 0c0b22ba92123e7d9e116aaef53ed7ea2897d28f